Running OMPL as a pre-processor for STOMP doesn't work.

### Description

I'm following the Planning Adapter Tutorials of Moveit! at http://docs.ros.org/kinetic/api/moveit_tutorials/html/doc/planning_adapters/planning_adapters_tutorial.html, but I can't implement the Running OMPL as a pre-processor for STOMP. Although I installed stomp_core from source which origined from https://github.com/ros-industrial/stomp_ros, I got an error that `stomp/SmoothingAdapter` doesn't exist.


### Your environment
* ROS Distro: [Kinetic]
* OS Version:  Ubuntu 16.04 LTS

### Steps to reproduce
just follow the tutorials and run :
`roslaunch panda_moveit_config demo.launch`

### Expected behaviour
The OMPL and STOMP works together to get a solution of the manipulator without error.

###  Console output

`[ERROR] [1581905685.671780997]: Exception while loading planning adapter plugin 'stomp/SmoothingAdapter': According to the loaded plugin descriptions the class stomp/SmoothingAdapter with base class type planning_request_adapter::PlanningRequestAdapter does not exist. Declared types are chomp/OptimizerAdapter default_planner_request_adapters/AddIterativeSplineParameterization default_planner_request_adapters/AddTimeOptimalParameterization default_planner_request_adapters/AddTimeParameterization default_planner_request_adapters/Empty default_planner_request_adapters/FixStartStateBounds default_planner_request_adapters/FixStartStateCollision default_planner_request_adapters/FixStartStatePathConstraints default_planner_request_adapters/FixWorkspaceBounds default_planner_request_adapters/ResolveConstraintFrames industrial_trajectory_filters/AddSmoothingFilter industrial_trajectory_filters/NPointFilter industrial_trajectory_filters/UniformSampleFilter`

--- a/doc/planning_adapters/planning_adapters_tutorial.rst
+++ b/doc/planning_adapters/planning_adapters_tutorial.rst
@@ -118,7 +118,7 @@ To achieve this, follow the steps:
                    default_planner_request_adapters/FixStartStateBounds
                    default_planner_request_adapters/FixStartStateCollision
                    default_planner_request_adapters/FixStartStatePathConstraints
-                   default_planner_request_adapters/STOMPSmoothingAdapter" />
+                   stomp_moveit/StompSmoothingAdapter" />
 
 #. The values of the ``planning_adapters`` is the order in which the mentioned adapters are called / invoked. Order here matters. Inside the STOMP adapter, a call to OMPL is made before invoking the STOMP smoothing solver, so STOMP takes the initial path computed by OMPL as the starting point to further optimize it.
 
@@ -154,7 +154,7 @@ To achieve this, follow the steps:
 
 #. Open the ``chomp_planning_pipeline.launch`` file in the ``<robot_moveit_config>/launch`` folder of your robot. For the Panda robot it is `this file <https://github.com/ros-planning/panda_moveit_config/blob/melodic-devel/launch/chomp_planning_pipeline.launch.xml>`_. Edit this launch file, find the lines where ``<arg name="planning_plugins">`` is mentioned and add the following lines below it: ::
 
-    <arg name="planning_adapters" value="default_planner_request_adapters/STOMPSmoothingAdapter" />
+    <arg name="planning_adapters" value="stomp_moveit/StompSmoothingAdapter" />
     <param name="request_adapters" value="$(arg planning_adapters)" />
 
 #. The values of the ``planning_adapters`` is the order in which the mentioned adapters are called / invoked. Order here matters. Inside the STOMP adapter, a call to CHOMP is made before invoking the STOMP smoothing solver, so STOMP takes the initial path computed by CHOMP as the starting point to further smoothen it.

